cli/flags.rs

```diff
@@ -422,6 +440,18 @@ pub fn flags_from_vec(args: Vec<String>) -> clap::Result<Flags> {
 
   let mut flags = Flags::default();
 
+  if matches.is_present("color") {
+    let preferece = matches.value_of("color").unwrap().to_string();
+    flags.color = match preferece.as_str() {
+      "auto" => ColorFlag::Auto,
+      "always" => ColorFlag::Always,
+      "never" => ColorFlag::Never,
+      _ => unreachable!(),
+    };
+    // DENO_COLOR env var is created to let runtime/color.rs know about the color flag
+    env::set_var("DENO_TERM_COLOR", preferece.clone());
```

Comment:
I'm unsure how reach the `--color` flag's value from `runtime/color.rs` and hence used env var, can this be improved ?